ArcGISRuntimeSDKQt_QMLSamples/UtilityNetwork/CreateLoadReport/CreateLoadReport.qml

```diff
@@ -0,0 +1,484 @@
+                } else {
+                    sampleStatus = 2;
+                    reportHasRun = true;
+                }
+            }
+        }
+
+        onErrorChanged: {
+            sampleStatus = -1;
+        }
+    }
+
+    UtilityCategoryComparison {
+        // Create a comparison to check the existence of service points.
+        id: serviceCategoryComparison
+        comparisonOperator: Enums.UtilityCategoryComparisonOperatorExists
+
+        // Service Category for counting total customers
+        function setServiceCategoryComparison() {
+            let utilityCategories = utilityNetwork.definition.categories;
+            for (let i = 0; i < utilityCategories.length; i++) {
+                if (utilityCategories[i].name === serviceCategoryName) {
+                    serviceCategoryComparison.category = utilityCategories[i];
+                    break;
+                }
+            }
+        }
+    }
+
+    UtilityTraceFunction {
+        id: addLoadAttributeFunction
+        functionType: Enums.UtilityTraceFunctionTypeAdd
+        condition: serviceCategoryComparison
+
+        function setNetworkAttribute() {
+            // The load attribute for counting total load.
+            addLoadAttributeFunction.networkAttribute = utilityNetwork.definition.networkAttribute(loadNetworkAttributeName);
+        }
+    }
+
+    UtilityTraceParameters {
+        id: traceParams
+        traceType: Enums.UtilityTraceTypeDownstream
+        startingLocations: [startingLocation]
+
+        Component.onCompleted: {
+            traceParams.resultTypes = [Enums.UtilityTraceResultTypeElements, Enums.UtilityTraceResultTypeFunctionOutputs]
+        }
+    }
+
+    function createUtilityAssetType() {
+        const networkDefinition = utilityNetwork.definition;
+        const networkSource = networkDefinition.networkSource(networkSourceName);
+        const assetGroup = networkSource.assetGroup(assetGroupName);
+        return assetGroup.assetType(assetTypeName);
+    }
+
+    function createUtilityTier() {
+        const networkDefinition = utilityNetwork.definition;
+        const domainNetwork = networkDefinition.domainNetwork(domainNetworkName);
+        return domainNetwork.tier(tierName);
+    }
+
+    function createStartingLocation() {
+        let loadTerminal;
+
+        // Get the terminal for the location. (For our case, use the "Load" terminal.)
+        const utilityTerminals = utilityAssetType.terminalConfiguration.terminals;
+
+        for (let i = 0; i < utilityTerminals.length; i++) {
+            if (utilityTerminals[i].name === terminalName) {
+                loadTerminal = utilityTerminals[i];
+                break;
+            }
+        }
+
+        if (!loadTerminal)
+            return;
+
+        return utilityNetwork.createElementWithAssetType(utilityAssetType, globalId, loadTerminal);
+    }
+
+    function createTraceConfiguration() {
+        let traceConfig = utilityTier.traceConfiguration;
```

Comment:
i think this can be const. in JS, const means it can't be reassigned, but it is mutable still